<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addnumber</class>
 <widget class="QDialog" name="addnumber">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="lab_room">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>55</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>room:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_type">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>85</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>type:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_addr">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>115</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>addr:</string>
   </property>
  </widget>
  <widget class="QLabel" name="lab_data">
   <property name="geometry">
    <rect>
     <x>70</x>
     <y>146</y>
     <width>67</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>data:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_room">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>50</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_type">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>80</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_addr">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>110</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="line_data">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>140</y>
     <width>113</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_add_member">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>210</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>add</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_back">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>210</y>
     <width>99</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>back</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
